connect/api/src/main/java/org/apache/kafka/connect/data/Values.java

```diff
@@ -70,7 +70,7 @@
     private static final String FALSE_LITERAL = Boolean.TRUE.toString();
     private static final long MILLIS_PER_DAY = 24 * 60 * 60 * 1000;
     private static final String NULL_VALUE = "null";
-    private static final String ISO_8601_DATE_FORMAT_PATTERN = "YYYY-MM-DD";
+    private static final String ISO_8601_DATE_FORMAT_PATTERN = "yyyy-MM-dd";
```

Comment:
hmm, is this just an underlying bug? should this bit be backported?

Reply:
yeah that's just a bug I happened to stumble upon, probably should be backported.